### ISSUE TITLE
not able to understand MAML in code files

### ISSUE THREAD

**Issue by tchavanr** [aioz-ai/MICCAI19-MedVQA#3]
As per mentioned in paper, for MAML: 5 tasks are considered for 1 iteration. Each task contains 3 classes out of 9 subcategories and 6 images (3 for train and 3 for validation). However inside the code sections, I am not finding this part. The validation set is not seen in the code, as only training is carried out. Also the meta update, train parts are not understandable. Can you please help to know in which file these things are considered and also a high level briefing on the code sections for MAML (meta learning).

**Comment by xuanbinh-nguyen96** [comment]
Hello,
- We do not provide source code MAML. You can use it directly from the author's official Repo via link: https://github.com/cbfinn/maml .
- The VQA_RAD dataset has no validation set.

**Comment by tchavanr** [comment]
The above MAML link is for the miniimagenet and omniglot dataset. The proposed work uses the meta learning for VQA_RAD dataset. how that is done and is it included in the current code? Or current code just handles the training of MEVF assuming the autoencoder and meta learning network are already trained. Or is it like the MAML network is trained offline and further is used as pre-trained network for MEVF and no meta trainng is conducted in MEVF training.

**Comment by xuanbinh-nguyen96** [comment]
- The MAML and CDAE networks are trained offline. After that, we use their trained weights to initialize the MEVF image feature extraction component in the VQA framework. We then finetune the whole VQA model using the training set of VQA-RAD dataset.
- Due to some restrictions in our company policy, the meta data (manual annotated data) used in this paper to learn meta model can not be shared yet. I will inform you later whenever I can share this dataset to you. So sorry for this problem!
